cns/configuration/configuration.go

```diff
@@ -133,6 +134,7 @@ func SetCNSConfigDefaults(config *CNSConfig) {
 	if config.ChannelMode == "" {
 		config.ChannelMode = cns.Direct
 	}
+	config.MetricsBindAddress = ":9090"
```

Comment:
It seems like this would always overwrite the address even if provided

Reply:
embarrassing